Firestore/Source/Public/FIRFieldValue.h

```diff
@@ -38,6 +38,27 @@ NS_SWIFT_NAME(FieldValue)
  */
 + (instancetype)fieldValueForServerTimestamp NS_SWIFT_NAME(serverTimestamp());
 
+/**
+ * Returns a sentinel that can be used with setData() or updateData() to union one or more
+ * elements into an array. Each specified element that doesn't already exist in the array will be
+ * added to the end. If the field being modified is not already an array it will be overwritten
+ * with an array containing exactly the specified elements.
+ *
+ * @param elements The elements to union into the array.
+ * @return The FieldValue sentinel for use in a call to setData() or updateData().
+ */
++ (instancetype)fieldValueForArrayUnion:(NSArray<id> *)elements NS_SWIFT_NAME(arrayUnion(_:));
```

Comment:
Please verify that this compiles correctly by changing `main.swift`.

Reply:
Woops. Thanks.